scripts/installer-lang/fi.ps1

```diff
@@ -0,0 +1,61 @@
+[PSCustomObject]@{
+    Welcome         = "
+╔══════════════════════════════╗
+║ Welcome to SpotX for Windows ║
```

Comment:
Will the logo remain in English or will you also translate it?

Reply:
I changed it.

Reply:
place text in the center

Reply:
Sorry, I didn't see that. Should be fixed now.